geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientUserAuths.java

```diff
@@ -90,28 +120,29 @@ public UserAuthAttributes getUserAuthAttributes(final Long userId) {
   }
 
   @VisibleForTesting
+  @TestOnly
   protected Collection<Subject> getSubjects() {
```

Comment:
Could this be "getAllSubjects" to make it clear that it's getting all subjects for all unique IDs?

Reply:
done